controllers/applicationvolumereplication_controller_test.go

```diff
@@ -269,3 +269,166 @@ var _ = Describe("ApplicationVolumeReplication controller", func() {
 		})
 	})
 })
+
+var _ = Describe("isManifestInAppliedState", func() {
+	timeOld := time.Now().Local()
```

Comment:
move these to line inside the dscribe in line 278

Reply:
The `timeOld` and `timeMostRecent` variables are inside each `Context` block as of `d41ab23fee18f7e5e248be22830fc12385e9075e`